cli/internal/kubecmd/kubecmd.go

```diff
@@ -500,6 +531,24 @@ func (k *KubeCmd) RemoveHelmKeepAnnotation(ctx context.Context) error {
 	return nil
 }
 
+type kubeDoer struct {
+	action func(ctx context.Context) error
+}
+
+func (k *kubeDoer) Do(ctx context.Context) error {
+	return k.action(ctx)
+}
+
+func retryFiveTimes(ctx context.Context, retryInterval time.Duration, action func(ctx context.Context) error, log debugLog) error {
```

Comment:
It looks strange to name this functions `retryFiveTimes` if the varianle defining the name is not inside the function and not named something like `fiveAttempts`. If one changes the `maxRetryAttempts` one also needs to rename this function. 
I'd probably just rename this function and pass the number of retries.